shell/platform/windows/dpi_utils.h

```diff
@@ -0,0 +1,23 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "Windows.h"
+
+#ifndef FLUTTER_SHELL_PLATFORM_WINDOWS_DPI_UTILS_H_
+#define FLUTTER_SHELL_PLATFORM_WINDOWS_DPI_UTILS_H_
+
+namespace flutter {
+
+/// Returns the current DPI. Supports all DPI awareness modes, and is backward
+/// compatible down to Windows Vista. If |hwnd| is nullptr, returns the DPI for
+/// the primary monitor. Otherwise, returns the system's DPI.
```

Comment:
The "Otherwise" here doesn't really make sense; it sounds like it's saying it'll return that if `hwnd` is not null (when it's actually the last-resort fallback, unless I'm misunderstand what you mean by "system DPI"). It's also odd that the comment never actually says that it returns the hwnd's DPI if possible.